u4-databases/views/uni-course-details.hbs

```diff
@@ -0,0 +1,8 @@
+{{#>uni iteration}}
```

Comment:
คุณเข้าใจวิธีการทำงานนี้หรือไม่? มันใหม่สำหรับฉัน :)
Do you understand how this works? It was new to me :)